### PR TITLE
update repo issue handling 

### LINKED ISSUE #5713: revise issues.rst documentation
[Link to document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/issues.rst). Mainly the roadmap section update after we are comfortable with changes to the roadmap handling.

I also propose to remove `status/review` labels and consider it a default state (i.e everything is ready for review unless it is marked `WIP` or `planning`). I think it is not worth the curation overhead.

### PR DESCRIPTION
**Pull request by martenson**
* make status/review the default state of a PR
* update the roadmap section to reflect reality

closes https://github.com/galaxyproject/galaxy/issues/5713

### CODE REVIEW

**Review by hexylena — APPROVED** [review]

File: doc/source/project/issues.rst [hunk 1]
```diff
@@ -215,17 +207,14 @@ special purpose.
...
-might have its own checklists to even more subcomponent cards, and so
-on.
+the project. This project will link to sub-issues and PRs which will go into
+much more detail, might link to other issues, projects, or PRs.
+
+This ``roadmap`` project is subject to periodic review every release.
 
-This ``roadmap`` issue will be assigned to every release milestone,
-forcing periodic review of the roadmap.
+Current roadmap project is `here <https://github.com/galaxyproject/galaxy/projects/8>`__.
```
Comment (nsoranzo) [inline doc/source/project/issues.rst]:
```suggestion
The current roadmap project is `here <https://github.com/galaxyproject/galaxy/projects/8>`__.
```

File: doc/source/project/issues.rst [hunk 2]
```diff
@@ -215,17 +207,14 @@ special purpose.
...
-We will maintain a single ``roadmap`` tagged meta-issue which will
+We will maintain a single ``roadmap`` GitHub project which will
 describe (at a very high level) the *current* major areas of focus for
-the project. This is similar to our PRIORITIES 2014/15 cards on Trello.
-Using `Task
-Lists <https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments>`__,
-this issue will link to sub-issues which will go into much more detail,
-might have its own checklists to even more subcomponent cards, and so
-on.
+the project. This project will link to sub-issues and PRs which will go into
```
Comment (nsoranzo) [inline doc/source/project/issues.rst]:
```suggestion
the project. This project will link to issues and PRs, which will go into
```

File: doc/source/project/issues.rst [hunk 3]
```diff
@@ -215,17 +207,14 @@ special purpose.
...
+much more detail, might link to other issues, projects, or PRs.
```
Comment (nsoranzo) [inline doc/source/project/issues.rst]:
```suggestion
much more detail and might link to other sub-issues, projects, or PRs.
```

**Review by nsoranzo — COMMENTED** [review]
The `Labeling Structure` intro still says that `status` is a required label.

**Review by jmchilton — APPROVED** [review]

**Comment by martenson** [comment]
updated, thanks for the feedback @nsoranzo !

File: doc/source/project/issues.rst [hunk 4]
```diff
@@ -58,12 +58,12 @@ Labeling Structure
...
 To allow for easy search, filtering, and general issue management every
-issue or PR (not tagged ``procedures`` or ``planning``) is *required* to
-have three labels which indicate the type, status, and focus area of the
-issue. Any issue without these three tags will be automatically have a
-``triage`` label applied indicating that it needs human intervention to
-be correctly tagged. These ``triage`` tagged issues will be regularly
-reviewed and tagged as appropriate.
+issue or PR (not tagged ``procedures`` or ``planning``) is expected to
+have two labels which indicate the type (``kind/``) and focus
+area(``area/``) of the issue. Any issue without these tags will
```
Comment (nsoranzo) [inline doc/source/project/issues.rst]:
```suggestion
area (``area/``) of the issue. Any issue without these tags will
```

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**